File: bin/run_queue.js
var sleep = require('sleep');
var shell = require('shelljs');

while(true){

  // shell.exec("time find /mnt/image/queue/wait/convert* | head -n 5 | parallel --load 60% /mnt/image/queue/jobqueue.sh {}");
  shell.echo('echo "hello" | head')
  sleep(300);

}

File: bin/run_queue.pl
#!/usr/bin/env perl

# my $cmd = "ls -lah |head";

my $cmd = "time find /mnt/image/queue/wait/convert* | head -n 10 | parallel --load 60% /mnt/image/queue/jobqueue.sh {}";
while(1){
  my @output = `$cmd`;
  print join("\n", @output)."\n"; 
  sleep(15);
}

